docs/authoring-components.md

```diff
@@ -2,51 +2,68 @@
 
 This document covers the guidelines on how to author new components, variants, and their properties within Figma. For questions and additional support, please reach out in the #primer channel in Slack.
 
+> **Edit 2022 May 13:** With the introduction of new component property features in Figma, the design systems team will be restructuring and renaming component/variant properties to closer align to the component APIs seen in code.
+
 ---
 
 ## General guidelines
 
-- Write component names in human speak. For example use `Button` instead of `btn`.
+- Write component names as reflected in code as a component name but not after utility names. For example use `Button` instead of `btn` .
```

Comment:
With this updated copy it sounds like `btn` refers to an utility name, which is not the case. `btn` is just an old way we reference the component name in Primer CSS.

Maybe this bullet point could also point to the [Spelling of component names ADR](https://github.com/github/primer/blob/main/adrs/2022-02-15-spelling-of-component-names.md).

Reply:
Thanks for the clarification. I will update the example and cross-reference the ADR.